students/pkoleyni/lesson04/trigram.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+import sys
+import random
+
+
+
+def make_list_of_words(line):
+    replace_reference = {ord('-'): ' ', ord(','): '', ord(','): '', ord('.'): '', ord(')'): '',  ord('('): '', ord ('"'): ''}
+    line = line.translate(replace_reference)
+    words = line.split()
+    return words
+
+def read_file (file_name):
+    big_line = []
+    with open(file_name, 'r') as f:
+        for line in f:
+            big_line.append(line)
+    return big_line
+
+
+def build_trigrams(words):
+    word_pairs = dict()
+    for i in range(len(words) - 2):
+        pair = tuple(words[i:i + 2])
+        follower = words[i + 2]
+        if pair not in word_pairs:
+            word_pairs[pair] = [follower]
+        else:
+            word_pairs[pair].append(follower)
+    return word_pairs
+
+
+def new_text(trigrams_dic):
+    a_list = []
+    l = len(trigrams_dic.keys())
+    for i in range(10):
+        random_number = random.randint(0,l)
+        key = list(trigrams_dic.keys())[random_number]
+        a_list.append(" ".join(list(key)))
+        a_list.append(" ".join(trigrams_dic[key]))
```

Comment:
you need to use a random here too to choose *ONE* word from your words associated with a key:
`word = random.choice(trigrams_dic[key])`